src/main/target/IFLIGHT_JBF7PRO/config.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of Cleanflight.
```

Comment:
could rename from CleanFlight to iNav in all files?

Reply:
This is what the manufacturer has provided, we can't just change the license text if their work was based on some older CF target or the likes.